### ISSUE TITLE
[BUG]: Extreme flickering in Neo Contra (SLUS-20961) opening cinematic

### ISSUE THREAD

**Issue by Tyestor** [PCSX2/pcsx2#6140]
### Describe the Bug

Extreme flickering in Neo Contra opening cinematic. No issue in software renderer, issue is present in OpenGL and D3D11. Cannot test D3D12 or Vulkan because my GPU is too old.

This flickering is bad enough that it will cause seizures in photosensitive epileptics (I am one of them, have to cover my eyes).

### Reproduction Steps

Fast/full boot game with qt ui, get past company logos, watch opening cutscene

### Expected Behavior

No flickering like in this youtube video: https://www.youtube.com/watch?v=azVh3H_EP6E

### PCSX2 Revision

PCSX2 Nightly - v1.7.2775 (Qt) SSE4

### Operating System

Windows 10 (64bit)

### CPU

i5 2500k

### GPU

Radeon HD6950

### GS Settings

default

### Emulation Settings

default

### GS Window Screenshots

**epilepsy warning**:

https://gfycat.com/threadbareunfoldedcirriped

### Logs & Dumps

i tried but i can't seem to get a gs dump in qt version.

**Comment by stenzek** [comment]
From a quick check the texcoords out out of range in the merge on the "borked" frames, but the input texture/source is fine.

Software is fine.

**Comment by Tyestor** [comment]
just checked a slightly older build "pcsx2-v1.7.2460-windows-64bit-SSE4" - issue still persists here (D3D11/OGL). the issue also happens in another game "monster attack" (SLES-51856) where on startup a company logo flickers a lot. software mode is fine here.

i also checked the 1.6.0 stable build - the issue is not present here for neo contra. both D3D11 and OGL work. however, monster attack is broken as far back as this stable build. it's possible its a different issue, although its a similar effect.

**Comment by stenzek** [comment]
> basically, it's two images, one at 0000, one at 1180 or whatever
> with conservative fb on, we cut the first one off at 512
> because it's 448 high, we say "well, that's a partial hit", and don't create a target at 1180
> even though it's only 64 pixels of the 448 it needs

tldr; disable conservative fb in game properties, or use software for fmvs, but we're looking into another possible solution which won't need it

**Comment by Tyestor** [comment]
Disabling conservative buffer allocation both fixed neo contra and monster attack. Thank you.

**Comment by Tyestor** [comment]
> > basically, it's two images, one at 0000, one at 1180 or whatever
> > with conservative fb on, we cut the first one off at 512
> > because it's 448 high, we say "well, that's a partial hit", and don't create a target at 1180
> > even though it's only 64 pixels of the 448 it needs
> 
> tldr; disable conservative fb in game properties, or use software for fmvs, but we're looking into another possible solution which won't need it

aethersx2 seems to have fixed this issue, but they confusingly have said that this isn't an issue in upstream?

https://www.patreon.com/posts/alpha-2226-june-67663677 (scroll to "Fix FMV flicker in some games (e.g. Neo Contra).")
